src/Plugins/PublishResultModel.cs

```diff
@@ -34,5 +34,10 @@ public class PublishResultModel
         /// Gets or sets a value containing errors that occurred when attempting to initiate publishing.
         /// </summary>
         public string PublishInitiateErrors { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value containing the full path of the published file.
+        /// </summary>
+        public string ResultFilePathName { get; set; }
```

Comment:
Is it a path or name? Comment says path? ResultFilePath or ResultFileName makes more sense?

Reply:
Also, I do not suggest passing full drive paths in any model across the network. We took out a bunch of that in Inspire. Security concern giving away full drive/path/name structure over the wire.

Reply:
I would be inclined to change it to ResultFilePath. This is used internally to move the file into place, but, it is then eventually used to pass the download url. The drive path never gets sent out over the wire